about.html

```diff
@@ -0,0 +1,157 @@
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+      integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+
+    <link rel="stylesheet" href="style.css" />
+    <title>GGP Restaurant Food and Drink</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <div class="nav">
+          <div class="nav_container">
+            <img src="./images/facebook.svg" alt="facebook-logo" class="icon"/>
+            <img src="./images/twitter.svg" alt="twitter-logo" class="icon"/>
+            <ul class="nav_menu">
+              <li class="nav_menu-item"><a href="#">English</a></li>
+              <li class="nav_menu-item"><a href="#">My Page</a></li>
+              <li class="nav_menu-item"><a href="#">Logout</a></li>
+            </ul>
+          </div>
+        </div>
+        <div class="bottom-nav">
+          <div class="bottom-nav_center hidden" id="menu">
+            <a class="logo" href="index.html">
+              <image src="./images/logo.png" alt="logo" class="icon-img"/>
+            </a>
+            <ul class="nav-list">
+              <li class="nav-list_item"><a href="index.html">Home</a></li>
+              <li class="nav-list_item"><a href="about.html">About Us</a></li>
+              <li class="nav-list_item"><a href="#chefs">Our Chef</a></li>
+              <li class="nav-list_item"><a href="#menus">Menu</a></li>
+              <li class="nav-list_item"><a href="#testimonials">Testimonials</a></li>
+              <li class="nav-list_item"><a href="#partners">Partners</a></li>
+              <li class="nav-list_item ggp-restaurant"><a href="#">GGP Food&Drink</a></li>
+            </ul>
+          </div>
+        </div>
+        <div class="bottom-nav_hamburger" id="hambuger">
+          <img src="./images/hamburger.svg" alt="hamburger" class="open ham-icon" id="openHamburger"/>
+          <img src="./images/closeicon.svg" alt="hambuger" class="close hidden ham-icon" id="closeHamburger"/>
+        </div>
+      </nav>
+      <div class="about_restaurant" id="about_restaurant">
+        <h3 class="about_restaurant-title">"A haven for food lovers, where every dish tells a unique story."</h3>
+        <h1 class="about_restaurant-heading">
+          Gyi Gyi Paw <br />
+          Food and Drink Restaurant
+        </h1>
+        <p class="about_restaurant-body">
+          "Welcome to Gyi Gyi Paw Restaurant, where culinary excellence meets a warm 
+          and inviting atmosphere.
+          Prepare to embark on a gastronomic journey that will delight your senses 
+          and leave a lasting impression.
+          At Gyi Gyi Paw Restaurant, our talented chefs craft a fusion of traditional 
+          and innovative dishes inspired by the rich culinary traditions of cuisine,
+          the finest local ingredients and expert craftsmanship."
+        </p>
```

Comment:
- [ ] Kindly consider making the background color of the text to be like the Behnace as you see here it is different from it ,so let's make it nice for the users 🚀

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Original design | Your design| 
| --- | --- |
|   ![image](https://github.com/YoutLone/Capstone-Project/assets/83360636/587793fd-9fb1-4205-bd94-a46b94baebcd)  |    ![image](https://github.com/YoutLone/Capstone-Project/assets/83360636/b858d0f0-a25d-4212-8434-5290e0049b1e)   |